.github/workflows/ci.yml

```diff
@@ -0,0 +1,71 @@
+name: Compile and run tests
+
+on: [push, pull_request]
+
+jobs:
+  test_standard_build:
```

Comment:
Could you drop this job, testing rpm build is part of our standard CI workflow now?

Reply:
Yup.